src/server/api/user.ts

```diff
@@ -11,7 +11,7 @@ import { logger } from '../config'
 import { redirectClient } from '../redis'
 import blacklist from '../resources/blacklist'
 import { isHttps, isValidShortUrl } from '../../shared/util/validation'
-import { generatePresignedUrl } from '../util/aws'
+import { PRIVATE, PUBLIC, generatePresignedUrl, setS3ObjectACL } from '../util/aws'
```

Comment:
consider exporting an `enum` object containing `PUBLIC`/`PRIVATE` instead?

Reply:
yes that was braindead of me to forget the existence of enums